trieste/acquisition/rule.py

```diff
@@ -294,3 +294,61 @@ def acquire(
         state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
 
         return point, state_
+
+
+class BatchAcquisitionRule(AcquisitionRule[None, SearchSpace]):
+    """ Implements a acquisition rule for a batch of query points """
+
+    def __init__(self, num_query_points: int,
+                 builder: Optional[AcquisitionFunctionBuilder] = None):
```

Comment:
this won't work for non-batch `AcquisitionFunction`s, and therefore also for `AcquisitionFunctionBuilder`s. If we're going to keep batch and non-batch separate, I assumed we'll need a new `BatchAcquisitionFunctionBuilder` type alias and `BatchAcquisitionFunction` interface.

Reply:
you can copy-paste the `AcquisitionFunction` and `AcquisitionFunctionBuilder` definition, and just add a bit about the shapes for `AcquisitionFunction` and `BatchAcquisitionFunction`. No tests needed

Reply:
could this be part of another PR?

Reply:
not really, cos you'll need to create a `BatchAcquisitionFunctionBuilder` to test this rule with